components/CharacterCard.vue

```diff
@@ -0,0 +1,50 @@
+<script lang="ts" setup>
+
+defineProps({
+  character: {
+    type: Object,
+    required: true
+  }
+});
+
+const bgByCharacterGender = (gender) => {
```

Comment:
follow the naming convention `getBgByCharacterGender`

![image](https://user-images.githubusercontent.com/29782224/185098352-2a750f44-4666-4249-baa4-a30ede2fc933.png)

gender argument should be typed, at least it should be `string`